install.sh

```diff
@@ -75,10 +75,10 @@ do
             shift
         ;;
         *)
-            if [[ ! -z "$prefix" ]]; then
+            if [[ -z "$prefix" ]]; then
                 if [ ! -d "$1" ]; then
                     set +e
-                    mkdir -p "$1" || help=1; break
```

Comment:
The "|| help=1; break" is usefull in the case of an unrecognized option. For example:
./install -oo output, will faill with a message
`mkdir: invalid option -- 'o'
Try 'mkdir --help' for more information.`

Maybe adding an extra case "-*)" is better for catching non-recognized options.

Reply:
> Maybe adding an extra case "-*)" is better for catching non-recognized options.

Good idea; the problem that I had is that using just the prefix path triggered the help message. I'll add the extra case.